plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/upgrade-notice.tsx

```diff
@@ -52,6 +53,9 @@ const UpgradeNotice = ( { revertMigration }: UpgradeNoticeProps ) => {
 
 	const handleRevert = () => {
 		revertMigration();
+		recordEvent( 'product_collection_block_transform', {
```

Comment:
I suggest we stick to the general convention: `wcadmin_blocks_<block>_<action>_<optional qualifier>` (more details: PdnLyh-6hh-p2).

What do you think about:

```suggestion
		recordEvent( 'blocks_product_collection_migration_between_products_beta', {
```

`block_transform` makes me think about transforming between collections rather than upgrade or migration from/to Products (Beta). 🤔

Reply:
Good idea!